src/include/basil.h

```diff
@@ -649,6 +735,13 @@ typedef struct basil_node_memory {
 	struct basil_node_memory *next;
 } basil_node_memory_t;
 
+/* This computeunit struct is added by Altair */
```

Comment:
PBS Pro is an open source project. The text of the copyright notice references Altair for legal reasons. There should be no references to Altair in other parts of the code. All contributors are equal, regardless of their employer.

If you want to state the reason for adding the structure (i.e. to support new BASIL syntax as of BASIL version X.Y), please do so.